CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 * Fixed `showName` warning by changing `title` property to `name` value in the controls addon.
+* Changed addons import module from `register` to `manager` for all addons.
```

Comment:
Please keep in mind that the newer change log should be on top of the existing one.